prac_05/emails.py

```diff
@@ -0,0 +1,31 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:  35 minutes
+"""
+
+
+def main():
+    """Print names from user emails."""
+    email_to_name = {}
+    user_email = input("Email: ")
+    while user_email != '':
+        name = get_name_from_email(user_email)
+        valid_name = input(f"Is your name {name.title()} (Y/n) ").lower()
```

Comment:
Could the variable called valid_name be misleading as the user isn't entering a name but a **selection**?